### DISCUSSION TITLE
Get ax from mouse hover event

### DISCUSSION THREAD

**Discussion by simsam8** [highfestiva/finplot#489] [Q&A] [answered]
Hi,

Is there any way to get the current axis from the mouse callback event?

**Comment by highfestiva** [comment, marked as answer]
Use [partial](https://docs.python.org/3/library/functools.html#functools.partial). Like so:
```python
fplt.set_mouse_callback(partial(my_callback, ax), ax=ax)
```

**Reply by simsam8** [reply]
Thanks!
